src/script/lua_api/l_mainmenu.cpp

```diff
@@ -1036,6 +1036,19 @@ int ModApiMainMenu::l_get_package_list(lua_State *L)
 		lua_pushstring(L, package.url.c_str());
 		lua_settable  (L, top_lvl2);
 
+		lua_pushstring(L, "screenshots");
+		lua_newtable(L);
+		{
+			int top_screenshots = lua_gettop(L);
+			for (size_t i = 0; i < package.screenshots.size(); ++i) {
+				lua_pushnumber(L, i + 1);
+				std::cerr <<  package.screenshots[i] << std::endl;
+				lua_pushstring(L, package.screenshots[i].c_str());
+				lua_settable(L, top_screenshots);
+			}
+		}
+		lua_settable  (L, top_lvl2);
```

Comment:
Space alignment looks wrong after `lua_newtable(L);` and the nine following lines in between.